cli/exchange/node.go

```diff
@@ -155,7 +156,14 @@ func NodeCreate(org, nodeIdTok, node, token, userPw, arch string, nodeName strin
 		if nodeType == "" {
 			nodeType = persistence.DEVICE_TYPE_DEVICE
 		}
-		putNodeReq := exchange.PutDeviceRequest{Token: nodeToken, Name: nodeName, NodeType: nodeType, SoftwareVersions: make(map[string]string), PublicKey: []byte(""), Arch: arch}
+
+		// Need to set cluster namespace for cluster node
+		var clusterNamespacObj *string
+		if nodeType == persistence.DEVICE_TYPE_CLUSTER {
+			clusterNamespace := cutil.GetClusterNamespace()
+			clusterNamespacObj = &clusterNamespace
+		}
+		putNodeReq := exchange.PutDeviceRequest{Token: nodeToken, Name: nodeName, NodeType: nodeType, ClusterNamespace: clusterNamespacObj, SoftwareVersions: make(map[string]string), PublicKey: []byte(""), Arch: arch}
```

Comment:
Setting cluster namespace in CLI is not needed. The setting should only happen in the agent when the node is registered.